### PR TITLE
feat(@clayui/table)!: `bordered` property was renamed to `borderedColumns` due consistency and `borderless` property was added (#2387)

### PR DESCRIPTION
**Pull request by diegonvs**
Fixes: #2387

### CODE REVIEW

**Comment by wincent** [comment]
FYI, I think the reason the Semantic Pull Request bot isn't giving you the green tick here is because of the commit message `feat(@clayui/table)!: `bordered` property was renamed to...`. It's actually a fine message, but the bot doesn't (yet) grok the exclamation point: https://github.com/probot/semantic-pull-requests/issues/51

**Comment by diegonvs** [comment]
Thanks for the information Greg! ;)
Resent just adding a `BREAKING CHANGE` message on commit's body.

**Comment by wincent** [comment]
@diegonvs: I didn't mean for you to change it — I just meant it is ok for you to ignore the bot because the bot is wrong. 😂

File: packages/clay-table/stories/index.tsx [hunk 1]
```diff
@@ -365,7 +366,8 @@ storiesOf('ClayTable', module)
 						{bottom: 'bottom', middle: 'middle', top: 'top'},
 						'middle'
 					)}
-					bordered={boolean('bordered', false)}
+					borderedColumns={boolean('bordered', false)}
```
Comment (matuzalemsteles) [inline packages/clay-table/stories/index.tsx]:
```suggestion
					borderedColumns={boolean('bordered columns', false)}
```

File: packages/clay-table/stories/index.tsx [hunk 2]
```diff
@@ -177,7 +177,8 @@ storiesOf('ClayTable', module)
 						{bottom: 'bottom', middle: 'middle', top: 'top'},
 						'middle'
 					)}
-					bordered={boolean('bordered', false)}
+					borderedColumns={boolean('borderedColumns', false)}
```
Comment (matuzalemsteles) [inline packages/clay-table/stories/index.tsx]:
```suggestion
					borderedColumns={boolean('bordered columns', false)}
```

**Pull request merged**